ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -135,7 +144,7 @@ Item {
 
             // HACK: we call `addNewMessagesMarker` later because messages model
             // may not be yet propagated with unread messages when this signal is emitted
-            if (chatLogView.visible && !d.isMostRecentMessageInViewport) {
+            if (chatLogView.visible) {
```

Comment:
@osmaczko I changed this condition to make sure the line appears even if chat was active. As you can see in the video, it behaves well. before, it wouldn't appear in that scneario.
Do you know if removing that condition could have unforeseen consequences?

Reply:
It will most likely misbehave, new messages marker should not be displayed when the chat is in focus/visible and new message arrives.

See behavior from 0:00 to 0:13 in the video https://github.com/status-im/status-desktop/pull/8570. And then from 0:13 to 0:23.

Reply:
Nice catch. You were right. I updated the condition to check if the app is active.

It looks like this now: 

[unread-line.webm](https://github.com/user-attachments/assets/2e424e26-04f7-4cda-823e-6129a02bad51)